app/views/expenses/edit.html.slim

```diff
@@ -0,0 +1,10 @@
+h1 Editing expense
+
+== render "form", expense: @expense
+
+br
+
+div
+  => link_to "Show this expense", @expense
+  '|
+  =< link_to "Back to expenses", expenses_path
```

Comment:
This doesn't look equivalent to the same file in chargings and repairs views https://github.com/Sf-60-Ruby-Cisco/BE-Ruby-Project/blob/main/app/views/repairs/edit.html.slim